libra_swarm/src/swarm.rs

```diff
@@ -230,20 +230,22 @@ impl LibraSwarm {
         let num_launch_attempts = 5;
         for i in 0..num_launch_attempts {
             info!("Launch swarm attempt: {} of {}", i, num_launch_attempts);
-            match Self::launch_swarm_attempt(
+
+            if let Ok(mut swarm) = Self::configure_swarm(
```

Comment:
Should we get rid of `launch_swarm` and replace it with `configure_swarm(...) -> Self` and `launch(&mut self )`?
That way, we don't lose the functionality to retry launch if something fails.